.github/workflows/pregenerate.yml

```diff
@@ -1,46 +1,48 @@
-name: Check Pregenerated Files
-
-on:
-  pull_request:
-  push:
-    branches-ignore:
-      - main
-
-concurrency:
-  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
-  cancel-in-progress: true
-
-jobs:
-  update:
-    name: "Update"
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-      - name: Set up Python 3.9
-        uses: actions/setup-python@v5
-        with:
-          python-version: 3.9
-      - name: Install jinja
-        run: python -m pip install jinja2
-      - name: Install protobuf dependencies
-        run: sudo apt-get update && sudo apt-get install -y protobuf-compiler && wget https://github.com/HebiRobotics/QuickBuffers/releases/download/1.3.3/protoc-gen-quickbuf-1.3.3-linux-x86_64.exe && chmod +x protoc-gen-quickbuf-1.3.3-linux-x86_64.exe
-      - name: Run hal
-        run: ./hal/generate_usage_reporting.py
-      - name: Run ntcore
-        run: ./ntcore/generate_topics.py
-      - name: Run wpimath
-        run: ./wpimath/generate_numbers.py && ./wpimath/generate_quickbuf.py protoc protoc-gen-quickbuf-1.3.3-linux-x86_64.exe
-      - name: Add untracked files to index so they count as changes
-        run: git add -A
-      - name: Check output
-        run: git --no-pager diff --exit-code HEAD
-      - name: Generate diff
-        run: git diff HEAD > pregenerated-files-fixes.patch
-        if: ${{ failure() }}
-      - uses: actions/upload-artifact@v4
-        with:
-          name: pregenerated-files-fixes
-          path: pregenerated-files-fixes.patch
-        if: ${{ failure() }}
+name: Check Pregenerated Files
```

Comment:
Looks like the line endings changed in this file.  They should be LF; if they're not, let's open a separate PR for that.